ROOT-RAMDISK/res/customconfig/customconfig.xml

```diff
@@ -311,28 +311,28 @@
 
 		<seekBar description="Set HotPlug Maximum temperature on Cores. When temperature reaches defined value, it turns OFF cores from 4 to (set in MAX ONLINE CORE option) in order to cool down the device. It can't be higher than CPU MAX HEAT Control. This option works only with Intelli CPU Cores Control." name="Intelli CORE Max Heat" action="generic /sys/module/msm_thermal/parameters/core_limit_temp_degC" unit="°C" min="50" reversed="false" step="1" max="80"/>
 
-		<seekBar description="Adjust the Core HotPlug Temperature Hysteresis for driver to calculate temperature differences and switching cores ON/OFF. Cores can be brought online if needed, works only with Intelli CPU Cores Control." name="Core Temperature Hysteresis" action="generic /sys/module/msm_thermal/parameters/core_temp_hysteresis_degC" unit="°C" min="5" reversed="false" step="1" max="20"/>
+		<seekBar description="Adjust the Core HotPlug Temperature Hysteresis for driver by calculating temperature differences and switching cores ON/OFF. Cores can be brought online if needed. Works only with Intelli CPU Cores Control." name="Core Temperature Hysteresis" action="generic /sys/module/msm_thermal/parameters/core_temp_hysteresis_degC" unit="°C" min="5" reversed="false" step="1" max="20"/>
 
 	</settingsPane>
 
 </settingsTab>
 
-<settingsTab name="CPU HOTPLUG">
+<settingsTab name="CPU HotPlug">
 
-	<settingsPane description="Setup CPU HotPlug settings and CPU Governors." name="Global HOTPLUG">
+	<settingsPane description="Setup CPU HotPlug settings and CPU Governors." name="Global HotPlug">
 
-		<spinner description="Switch from OLD LG Default Hot Plug driver to MSM Hot Plug, Alucard or Intelligent Hot Plug driver. Note that using Intelligent CPU Thermal Control with Default LG HotPlug is NOT recommended (causing low performance)." name="Hot Plug Driver" action="hotplug">
+		<spinner description="Switch from OLD LG Default HotPlug driver to other custom HotPlug drivers. Note that using Intelligent CPU Thermal Control with Default LG HotPlug is NOT recommended (causing low performance)." name="Hot Plug Driver" action="hotplug">
 			<spinnerItem name="Default LG" value="default"/> 
-			<spinnerItem name="MSM Hot Plug" value="msm_hotplug"/> 
-			<spinnerItem name="AutoSMP Hot Plug" value="autosmp"/> 
-			<spinnerItem name="Alucard Hot Plug" value="alucard"/> 
-			<spinnerItem name="Thunder Hot Plug" value="thunderplug"/> 
-			<spinnerItem name="Intelligent Hot Plug" value="intelli"/> 
+			<spinnerItem name="MSM HotPlug" value="msm_HotPlug"/> 
```

Comment:
value="msm_HotPlug"/>  == WRONG! XML CRASH, again i need to search why.
dont use search and replace or copy paste. this is not TEXT, this is critical config.